client/app/scripts/components/node-details/node-details-table-node-metric-link.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import CloudLink from '../cloud-link';
+import { formatMetric } from '../../utils/string-utils';
+import { trackMixpanelEvent } from '../../utils/tracking-utils';
+import { TABLE_VIEW_MODE } from '../../constants/naming';
+
+class NodeDetailsTableNodeMetricLink extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick() {
+    trackMixpanelEvent('scope.node.metric.click', {
+      layout: TABLE_VIEW_MODE,
```

Comment:
`layout: TABLE_VIEW_MODE` refers to the overall layout for the scope canvas, as opposed to the table inside the details panel